doc/en/api/1.0.0/gwcglobal.yaml

```diff
@@ -6,8 +6,8 @@ info:
   description: Global configuration for the GeoWebCache server
   contact:
     name: GeoServer
-    email: 'geoserver-users@sourceforge.net'
-    url: 'http://geoserver.org/comm/'
+    email: 'geoserver-users@osgeo.org'
```

Comment:
Oh this is interesting, send email will not be accepted unless they have subscribed I don't think?

Reply:
Or should the email address be geoserver-user@discourse.osgeo.org?

![image](https://github.com/user-attachments/assets/867763d5-75f9-4606-98b6-c6118c38b19d)

Reply:
I just tested by sending an email to [geoserver-user@discourse.osgeo.org](mailto:geoserver-user@discourse.osgeo.org) which is not what we expected.

In earlier testing they setup a simple address (for the marketing committee). I would like to ask them to do the same here.

But yes we need to establish what this is intended to be :)